packages/edit-site/src/components/page-templates-template-parts/index.js

```diff
@@ -374,7 +374,7 @@ export default function PageTemplatesTemplateParts( { postType } ) {
 				data: filteredData,
 				view,
 				fields,
-				textFields: [ 'title' ],
+				textFields: [ 'title', 'author' ],
```

Comment:
I think this can work for now, but we could follow-up to do the following:

- add `text` type to fields
- remove `textFields` from `sortByTextFields`

Reply:
Probably `sortByTextFields` can become `sortByField` with more type field info.

Reply:
It's not that straightforward right now because `author` is an `type:enumeration`. Maybe we could explore if we could use `getValue` for those fields too, but we would need to think of what use cases we could have..

Reply:
I agree with removing `textFields`. It's true that not all enums behave like regular text fields, but we can approach this more generally. Also, is sort-by-author currently sorting by the _username_? What about display name?

Reply:
For templates and template parts we use the [author_text](https://github.com/WordPress/gutenberg/blob/trunk/lib/compat/wordpress-6.5/rest-api.php#L81), which is a custom implementation due to the different kind of authors. For example for theme items the actual persisted value in db is `author:0`, but we extract the theme name as author.